specs/beacon-chain.md

```diff
@@ -44,27 +43,38 @@ The primary source of load on the beacon chain are "attestations". Attestations
 | `WITHDRAWAL_PERIOD` | 2**19 (= 524,288) | slots | ~97 days |
 | `BASE_REWARD_QUOTIENT` | 2**15 (= 32,768) | — |
 | `MAX_VALIDATOR_CHURN_QUOTIENT` | 2**5 (= 32) | — | 
+| `LOGOUT_MESSAGE` | `"LOGOUT"` | — | 
```

Comment:
Should the string be something a little... stranger so that the validator would not have signed the message otherwise?

Reply:
Right, and now that I think about it we also need versioning here to avoid a replay attack. cc @vbuterin

Reply:
Maybe we make the message `"LOGOUT" + str(slot_height)` and include the `slot_height` in the logout `SpecialRecord`. The record can only be included at or after the specified slot height.  

We did something like this in the FFG contract.

Reply:
I meant versioning with things like `fork_version` so that a logout message on one chain cannot be replayed on a forked chain. Instead of doing it on a case-by-case basis for attestations and specials separately, maybe there's an opportunity to have unified middleware logic for versioning.

Reply:
I'm going to merge this and make an issue related to need for versioning in logouts

Reply:
Issue here